src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const baseCost = 40 * days;
+
+  if (days >= 7) {
+    return baseCost - 50;
+  } else if (days >= 3) {
+    return baseCost - 20;
```

Comment:
Replace the magic number `20` with a named constant, such as `DISCOUNT_FOR_THREE_DAYS`, to enhance code clarity.